onmt/utils/statistics.py

```diff
@@ -16,11 +16,12 @@ class Statistics(object):
     * elapsed time
     """
 
-    def __init__(self, loss=0, n_words=0, n_correct=0):
+    def __init__(self, loss=0, n_words=0, n_correct=0, computed_stats={}):
```

Comment:
Maybe `computed_metrics` to be more explicit (and consistent with the previous comment about `opts.scores`) ?